src/bp-templates/bp-nouveau/js/buddypress-activity.js

```diff
@@ -1356,6 +1356,21 @@ window.bp = window.bp || {};
 				target.closest( '.acomment-display' ).addClass( 'display-focus' );
 				target.closest( '.comment-item' ).addClass( 'comment-item-focus' );
 
+				var activity_data_nickname;
+				var current_user_nickname;
+
+				if ( ! _.isNull( activity_comment_data ) ) {
+					activity_data_nickname = activity_comment_data.nickname;
+				}
+
+				var atWho = '<span class="atwho-inserted" data-atwho-at-query="@" contenteditable="false">@' + activity_data_nickname + '</span>&nbsp;';
+
+				if ( ! _.isUndefined( BP_Nouveau.activity.params.user_domain ) ) {
+					var user_domain = BP_Nouveau.activity.params.user_domain;
+					var segments = user_domain.split( '/' );
+					current_user_nickname = segments[segments.length - 1] || segments[segments.length - 2];
```

Comment:
@annrra We can not use domain to fetch nickname because we provide to change `Link Format` username to unique identifier. Instead of this you can pass `activity_comment_user_id` in the localize var. Also you can get current user id from activity localize param and match that.